low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -461,3 +468,56 @@ def get_nw_status(self, nw_interface, interface):
             nw_cable_conn_status = "UNKNOWN"
             logger.exception(self.log.svc_log(err))
         return nw_status, nw_cable_conn_status
+
+    def get_disks_info(self):
+        """Update and return server drive information in specific format."""
+        units_factor_GB = 1000000000
+        overall_usage = {
+                "totalSpace": f'{int(psutil.disk_usage("/")[0])//int(units_factor_GB)} GB',
+                "usedSpace": f'{int(psutil.disk_usage("/")[1])//int(units_factor_GB)} GB',
+                "freeSpace": f'{int(psutil.disk_usage("/")[2])//int(units_factor_GB)} GB',
+                "diskUsedPercentage": psutil.disk_usage("/")[3],
+            }
+        disks = []
+        for disk in Disks().get_disks():
+            uid = disk.path if disk.path else disk.id
+            disk_health = self.get_health_template(uid, True)
+            health_data = disk.get_health()
+            health = "OK" if health_data['SMART_health'] else "Fault"
+            self.set_health_data(disk_health, health, specifics=[{"SMART": health_data}])
+            disk_health["last_updated"] = int(time.time())
+            disks.append(disk_health)
+        disks_health_info = [{
+            "overall_usage": overall_usage,
+            "disk_count": len(disks),
+            "disks": disks,
+            "last_updated": int(time.time())
+        }]
+        return disks_health_info
+
+    def get_psu_info(self):
+        """Update and return PSU information in specific format."""
+        psus = []
+        psu_mapping = {
+            "PS1 Status": "PSU1",
+            "PS2 Status": "PSU2"
+        }
+        response, _, _ = SimpleProcess(shlex.split('ipmitool sdr type "Power Supply" -c')).run()
+        uids = {}
+        for psu in response.decode().strip("\n").split("\n"):
+            psu_name, sensor, *_ = psu.split(",")
+            uids[psu_mapping[psu_name]] = f"#0x{sensor.strip('h').lower()}"
+        response, _, _ = SimpleProcess("dmidecode -t 39").run()
+        matches = re.finditer(r"System Power Supply\n(\s*.*\n)"
```

Comment:
We have so many fields from dmidecode, why we are not using these 
>System Power Supply
        Location: PSU1
        Name: PWS-751P-1R
        Manufacturer: SUPERMICRO
        Serial Number: P751PCJ32CU1284
        Revision: 1.1
        Max Power Capacity: 750 W
        Status: Present, OK
        Type: Switching
        Input Voltage Range Switching: Auto-switch
        Plugged: Yes
        Hot Replaceable: Yes

Reply:
Used below fields now

```
       "Power Unit Group": "2",
        "Location": "PSU2",
        "Name": "PWS-751P-1R",
        "Serial Number": "P751PCJ35CU5374",
        "Max Power Capacity": "750 W",
        "Status": "Present, OK",
        "Plugged": "Yes",
        "Hot Replaceable": "Yes"
```